oms-automation-solution/README.md

```diff
@@ -0,0 +1,40 @@
+# OMS Azure Automation Job Monitoring
+
+
+
+This solution (currently in private preview) will allow you to visualize your Azure Automation jobs in Operations Management Suite (Log Analytics). This solution use the job status and job stream information provided by Microsoft (https://docs.microsoft.com/en-us/azure/automation/automation-manage-send-joblogs-log-analytics) to present data about your Azure Automation jobs in Log Analytics. 
+
+![alt text](images/AutomationJobs.PNG "Azure Automation Job Monitoring")
```

Comment:
the uri is case sensitive, so it must be changed. Suggest to rename "Images" folder to "images"